src/components/UI/Charts/Sunburst/InfoPanel.tsx

```diff
@@ -29,11 +31,36 @@ const splitTitle = (title: string) => {
   const [rawTitle, rawCode] = title.split('(HP:');
   return {
     title: rawTitle.trim(),
-    hpCode: `HP ${rawCode.replace(')', '').trim()}`,
+    hpCode: rawCode ? `HP ${rawCode.replace(')', '').trim()}` : '',
   };
 };
 
-const InfoPanel: FunctionComponent<Props> = ({ data, onClickAddTermToActiveIndex, treeData }) => {
+export const getPath = (node: string, treeNodes: TreeNode[], path: string[] = []) => {
+  const currentNodeText = treeNodes[0].key;
+  path.push(currentNodeText);
```

Comment:
here, the param is mutated. If possible, I would make a shallow copy of "path" (concat, slice, ...)

Reply:
in this case i think it ok that its mutated. The initial value is [] anyway.